browserify.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var webkitAssign = require('./index');
+
+module.exports = function (file) {
+	return webkitAssign();
+};
```

Comment:
Gotcha. Here's my wishlist:
* Remove the unused parameter
* Add a comment documenting what you just said
* Use 4 spaces instead of tabs (line 6)

Thanks!